community/libcanberra/APKBUILD

```diff
@@ -71,4 +65,11 @@ gtk3() {
 		"$subpkgdir"/usr/lib/gnome-settings-daemon-3.0/gtk-modules/
 }
 
+pulse() {
```

Comment:
Would it make sense to add an `isntall_if=` here so that it gets automatically installed when you have pulseaudio installed?

Reply:
Added